src/light_client.rs

```diff
@@ -57,7 +58,13 @@ pub async fn run(
 					}
 					let commitment = header.extrinsics_root.commitment.clone();
 
-					let positions = rpc::generate_random_cells(max_rows, max_cols);
+					let cell_count: u32;
+					if confidence == 100f64 {
+						cell_count = -((1f64 - 99.9f64 / 100f64).log2()).ceil() as u32;
```

Comment:
Should we increase precision in this case? Or do we want a conservative default?